admin_guide/aws_cluster_labeling.adoc

```diff
@@ -0,0 +1,45 @@
+[[admin-guide-aws-cluster-labeling]]
+= Cluster Labeling for Amazon Web Services (AWS)
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+This topic describes how to label an existing {product-title} cluster running on
+Amazon Web Services (AWS).
+
+In order to correctly identify which resources are associated to a cluster, resources
+should be tagged with the key `KubernetesCluster`.  Tagging all resources with the
```

Comment:
I'm fine with the newer labeling style so long as that is what the installer will check for and ensure is set.  The source material was updating KubernetesCluster.  Is the installer checking for the new or old style?

Reply:
For environments where the hosts are pre-provisioned we're merely checking that they've told us a cluster id. The installer has never required that variable be set previously so this will halt clean 3.7 installs and upgrades with a reference to this documentation so they can address the issue then set the variable as a signal to us that it's safe to proceed.

For environments where openshift-ansible provisions the hosts we tag using the new style.

Reply:
OK, then for consistency is makes sense to use the new style for these docs.  I will update